src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -668,6 +668,8 @@ void SizeZoneEquipment(EnergyPlusData &state)
             if (state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum) > 0.0) {
                 state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolTstatTemp =
                     state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum);
+                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatTstatTemp =
+                    state.dataHeatBalFanSys->ZoneThermostatSetPointLo(ActualZoneNum);
```

Comment:
I have a question. If TempZoneThermostatSetPoint > 0, and CoolTstatTemp = (is set equal to) TempZoneThermostatSetPoint (line 669 and 670), then why would HeatTstatTemp be set = to ZoneThermostatSetPointLo ? What if ZoneThermostatSetPointLo = 0 ? It hasn't been tested for being > 0. Why not set it to TempZoneThermostatSetPoint ? which is already tested for > 0. Also, you only made 1 change here, that gives me pause given the multiple possible load results. If (heatingload > 0), else take care of everything else.

I realize you are following this existing code. Where ZoneThermostatSetPointLo is used if TempZoneThermostatSetPoint = 0. But here, TempZoneThermostatSetPoint > 0.

        if (state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatLoad > 0.0) {
            state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatZoneRetTemp = RetTemp;
            if (state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum) > 0.0) {
                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatTstatTemp =
                    state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum);
            } else {
                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatTstatTemp =
                    state.dataHeatBalFanSys->ZoneThermostatSetPointLo(ActualZoneNum);
            }
        } else {
            state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolZoneRetTemp = RetTemp;
            if (state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum) > 0.0) {
                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolTstatTemp =
                    state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ActualZoneNum);
                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).HeatTstatTemp =
                    state.dataHeatBalFanSys->ZoneThermostatSetPointLo(ActualZoneNum);   <<<<<<<
            } else {
                state.dataSize->CalcZoneSizing(state.dataSize->CurOverallSimDay, ControlledZoneNum).CoolTstatTemp =
                    state.dataHeatBalFanSys->ZoneThermostatSetPointHi(ActualZoneNum);
            }
        }

This is my first pass at this and was rather quick. Just pointing out what I saw on this pass.

Reply:
Thanks for doing a quick initial pass on this.  I see your reasoning with TempZoneThermostatSetPoint and there are some additional situations I need to test for.  I did try using this variable and the Thermostat Setpoint Temperature at Peak Load results in a somewhat random value that doesn't match any setpoint values.

![image](https://user-images.githubusercontent.com/17144687/116913460-bc11da80-ac06-11eb-979d-d090d522bd01.png)

My thinking was adding these lines in the else statement, it basically provides a default for this value that matches the heating setpoint value since it is covering when HeatLoad or CoolLoad is equal to 0.  Maybe there is a better way to achieve this however as I was trying to following existing code.  Is it better to have a separate if statement for if HeatTstatTemp is equal to 0?  This does seem to achieve the same thing.  I can discuss with Neal in more detail this week as well.